app/src/main/java/com/android/voyageur/ui/trip/photos/PhotosScreen.kt

```diff
@@ -0,0 +1,313 @@
+        Image(
+            painter = rememberAsyncImagePainter(photoUri),
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            modifier = Modifier.fillMaxSize())
+      }
+}
+
+/**
+ * Composable function to display a dialog for viewing a full-size photo.
+ *
+ * @param photoUri The URI of the photo being displayed.
+ * @param photoList The list of all photos associated with the trip.
+ * @param onDismiss The callback to dismiss the dialog.
+ * @param initialIndex The initial index of the photo in the list.
+ * @param tripsViewModel The [TripsViewModel] to manage photo actions like deletion.
+ */
+@OptIn(ExperimentalCoroutinesApi::class)
+@Composable
+fun PhotoDialog(
+    photoUri: String,
+    photoList: List<String>,
+    onDismiss: () -> Unit,
+    initialIndex: Int,
+    tripsViewModel: TripsViewModel
+) {
+  var currentIndex by remember { mutableStateOf(initialIndex) }
+  val currentPhoto = photoList.getOrNull(currentIndex)
+  val context = LocalContext.current
+  var showDialog by remember { mutableStateOf(false) }
+  val status by connectivityState()
+  val isConnected = status === ConnectionState.Available
+
+  Box(
+      modifier =
+          Modifier.fillMaxSize()
+              .background(Color.Black.copy(alpha = 0.8f))
+              .clickable { onDismiss() }
+              .testTag("photoDialog_${photoUri}"),
+      contentAlignment = Alignment.Center) {
+        currentPhoto?.let {
+          Column(
+              horizontalAlignment = Alignment.CenterHorizontally,
+              verticalArrangement = Arrangement.Center,
+              modifier = Modifier.wrapContentSize()) {
+                Box(modifier = Modifier.fillMaxWidth().wrapContentSize(align = Alignment.Center)) {
+                  // Image
+                  Image(
+                      painter = rememberAsyncImagePainter(it),
+                      contentDescription = stringResource(R.string.full_size_photo),
+                      contentScale = ContentScale.Fit,
+                      modifier = Modifier.fillMaxWidth().padding(16.dp).clickable {})
+
+                  // Row to position the left and right buttons at the middle of the height
+                  Row(
+                      horizontalArrangement = Arrangement.SpaceBetween,
+                      verticalAlignment = Alignment.CenterVertically,
+                      modifier =
+                          Modifier.fillMaxWidth()
+                              .align(Alignment.Center) // Center the Row vertically
+                      ) {
+                        // Left Button ("<")
+                        IconButton(
+                            modifier = Modifier.testTag("goLeftButton"),
+                            onClick = {
+                              currentIndex = (currentIndex - 1 + photoList.size) % photoList.size
+                            },
+                            enabled = photoList.size > 1) {
```

Comment:
Good feature! But the left / right arrows don't really look good I think instead of using the `<` and `>` symbols you could use material3 icons for a better UX

Reply:
You're right, I changed the icons and I will attach a screenshot with the updated UI.
![Screenshot 2024-12-08 143846](https://github.com/user-attachments/assets/b51e3232-d9bd-4b7b-a74d-73931e333992)